docs/docs/configuration/auth.md

```diff
@@ -146,12 +146,15 @@ If you are using GitHub enterprise, make sure you set the following to the appro
 
 ### Keycloak Auth Provider
 
-1.  Create new client in your Keycloak with **Access Type** 'confidental' and **Valid Redirect URIs** 'https://internal.yourcompany.com/oauth2/callback'
+1.  Create new client in your Keycloak realm with **Access Type** 'confidental' and **Valid Redirect URIs** 'https://internal.yourcompany.com/oauth2/callback'
 2.  Take note of the Secret in the credential tab of the client
 3.  Create a mapper with **Mapper Type** 'Group Membership' and **Token Claim Name** 'groups'.
 
+NOTE: this is the legacy Keycloak Auth Prodiver, use `keycloak-oidc` if possible.
```

Comment:
This will highlight it as a note in the rendered documentation
```suggestion
:::note this is the legacy Keycloak Auth Prodiver, use `keycloak-oidc` if possible. :::
```